CrowdStrike/crowdstrike-telemetry/tests/telemetry_event_36.json

```diff
@@ -0,0 +1,42 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "CrowdStrike Telemetry",
+        "dialect_uuid": "22f2afd2-c858-443d-8e06-7b335e439c29"
+      }
+    },
+    "message": "{\"AgentLoadFlags\":\"1\",\"AgentLocalTime\":\"1688462537.0\",\"AgentTimeOffset\":\"-5574.207\",\"AgentVersion\":\"6.54.16812.0\",\"BiosManufacturer\":\"HP\",\"BiosVersion\":\"U21 Ver. 02.05.00\",\"ChassisType\":\"Mini PC Desktop\",\"City\":\"Toronto\",\"ComputerName\":\"DEFP0395\",\"ConfigBuild\":\"1007.3.0016812.10\",\"ConfigIDBuild\":\"16812\",\"Continent\":\"America\",\"Country\":\"Canada\",\"FalconGroupingTags\":\"none\",\"FirstSeen\":\"1688132251\",\"HostHiddenStatus\":\"Visible\",\"MachineDomain\":\"example.org\",\"OU\":\"Computers;1001.example.toronto;Workspace;Canada;Country\",\"PointerSize\":\"8\",\"ProductType\":\"1\",\"SensorGroupingTags\":\"none\",\"ServicePackMajor\":\"0\",\"SiteName\":\"EQX\",\"SystemManufacturer\":\"HP\",\"SystemProductName\":\"HP Elite Mini 800 G9 Desktop PC\",\"Time\":\"1688542167.026\",\"Timezone\":\"America/New York\",\"Version\":\"Windows 10\",\"aip\":\"1.2.3.4\",\"aid\":\"111111111111111\",\"cid\":\"222222222222222\",\"event_platform\":\"Win\"}"
+  },
+  "expected": {
+    "message": "{\"AgentLoadFlags\":\"1\",\"AgentLocalTime\":\"1688462537.0\",\"AgentTimeOffset\":\"-5574.207\",\"AgentVersion\":\"6.54.16812.0\",\"BiosManufacturer\":\"HP\",\"BiosVersion\":\"U21 Ver. 02.05.00\",\"ChassisType\":\"Mini PC Desktop\",\"City\":\"Toronto\",\"ComputerName\":\"DEFP0395\",\"ConfigBuild\":\"1007.3.0016812.10\",\"ConfigIDBuild\":\"16812\",\"Continent\":\"America\",\"Country\":\"Canada\",\"FalconGroupingTags\":\"none\",\"FirstSeen\":\"1688132251\",\"HostHiddenStatus\":\"Visible\",\"MachineDomain\":\"example.org\",\"OU\":\"Computers;1001.example.toronto;Workspace;Canada;Country\",\"PointerSize\":\"8\",\"ProductType\":\"1\",\"SensorGroupingTags\":\"none\",\"ServicePackMajor\":\"0\",\"SiteName\":\"EQX\",\"SystemManufacturer\":\"HP\",\"SystemProductName\":\"HP Elite Mini 800 G9 Desktop PC\",\"Time\":\"1688542167.026\",\"Timezone\":\"America/New York\",\"Version\":\"Windows 10\",\"aip\":\"1.2.3.4\",\"aid\":\"111111111111111\",\"cid\":\"222222222222222\",\"event_platform\":\"Win\"}",
```

Comment:
In this event, there are some interesting information that are not extracted:
- ComputerName -> `host.name`
- MachineDomain -> `host.domain`

and define `event.category = ['host']` and `event.type = ['info']`